### PR TITLE
New java command

### PR DESCRIPTION
**Pull request by vovapolu**
This PR will add new `java` command. See more in https://github.com/ensime/ensime-sbt/issues/196

### CODE REVIEW

File: src/sbt-test/sbt-big-project/run-java/project/runJavaBuild.scala [hunk 1]
```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2015 Sam Halliday
```
Comment (fommil) [inline src/sbt-test/sbt-big-project/run-java/project/runJavaBuild.scala]: your name goes here

File: src/sbt-test/sbt-big-project/run-java/test [hunk 2]
```diff
@@ -0,0 +1,9 @@
+> run-java/compile
+> run-java/runJava testing_key1=value1 testing_key2=value2 -Dtesting_key3=value3 -Dtesting_key3=value3 -Xms2G -Xmx2G runjava.printArgs output_args1 -arg1 -arg2
```
Comment (fommil) [inline src/sbt-test/sbt-big-project/run-java/test]: :tada:

File: src/sbt-test/sbt-big-project/run-java/project/runJavaBuild.scala [hunk 3]
```diff
@@ -1,4 +1,4 @@
-// Copyright (C) 2015 Sam Halliday
+// Copyright (C) 2015 Vladimir Polushin
```
Comment (fommil) [inline src/sbt-test/sbt-big-project/run-java/project/runJavaBuild.scala]: Awesome, did you finish the time machine too? :-P

Comment (vovapolu) [inline src/sbt-test/sbt-big-project/run-java/project/runJavaBuild.scala]: In progress, now it's able to move you only one year back ;)

**Pull request closed without merging**